modules/luigi/__init__.py

```diff
@@ -99,7 +102,8 @@ def yesno(ctx: rs.ContextWrapper):
         signal=sig_start_order_question,
         emit_detached=True)
     def prompt_order(ctx: rs.ContextWrapper):
-        ctx[rawio.prop_out] = "guess what! i'm selling ice cream. want some?"
+        out = verbaliser.get_random_question('greet_general')
+        ctx[rawio.prop_out] = out
```

Comment:
why do you use `out` here? I think you can write this in one line (like you did below)

Reply:
You are right, I forgot it while I am testing the code. :)